README.md

```diff
@@ -339,6 +340,51 @@ app({
 [View online](https://hyperapp-simple-routing.gomix.me/)
 </details>
 
+#### lifecycle events
+
+Hyperapp provides lifecycle events for the virtual HTML nodes:
+
+```js
+app({
+  view: _ => html`
+    <div oncreate=${e => console.log(e)}>Hi.</div>`
+})
+```
+
+The supported events are:
+
+- `oncreate(e : HTMLElement)`
+- `onupdate(e : HTMLElement)`
+- `onremove(e : HTMLElement)`
+
+The lifecycle event handler receives a reference to the DOM element.
+
+<details>
+<summary><i>Example</i></summary>
+
+```js
+function repaint(canvas, model) {
+  var ctx = canvas.getContext('2d')
```

Comment:
Could you please spell this out, `context` i know its a pain, but to most people ctx means nothing! Love the contrib though! :heart: